appendix-basic65.tex

```diff
@@ -7462,12 +7458,14 @@ \subsection{RSPCOLOR}
 \item [Format:] {\bf RSPCOLOR(n)}
 \item [Usage:]  Returns multi-colour sprite colours.
 
-                {\bf n} 1 : get multi-colour \# 1.
-
-                {\bf n} 2 : get multi-colour \# 2.
+                {\bf n} sprite number:
```

Comment:
`sprite number` doesn't sound quite right, as it then sounds like an index number to a sprite.

Perhaps instead, it could read as `sprite multi-colour number:`?

Reply:
I misunderstood the command. I admit, im not running the same programs, im purely just proof reading it.